src/Translatable/Translatable.php

```diff
@@ -458,4 +460,67 @@ public function __isset($key)
     {
         return $this->isTranslationAttribute($key) || parent::__isset($key);
     }
+
+    public function getTranslationChanges(): array
+    {
+        $translationChanges = [];
+        foreach ($this->translations as $translation) {
+            $changes = $translation->getChanges();
+            if (! empty($changes)) {
+                $translationChanges[$translation->locale] = $changes;
+            }
+        }
+
+        return $translationChanges;
+    }
+
+    public function wasTranslationChanged($attributes = null, $locale = null): bool
+    {
+        $translationChanges = $this->getTranslationChanges();
+        if (empty($translationChanges)) {
+            return false;
+        }
+        if (empty($attributes)) {
+            return true;
+        }
+        if (! is_array($attributes)) {
+            $attributes = [$attributes];
+        }
+        if (config('translatable.rule_factory.format') === RuleFactory::FORMAT_KEY) {
+            $attributes = array_map(function ($attribute) {
+                return implode('.', array_reverse(explode(':', $attribute)));
+            }, $attributes);
+        }
```

Comment:
As the user can provide attribute and locale I don't see a reason why we should do this "complex" string handling here. `wasTranslationChanged('title', 'de')` should check if the german title was changed. So it's easier to use/read and more clear than `wasTranslationChanged('de.name')` or `wasTranslationChanged('name:de')`.
By removing this string exploding the following logic also gets a lot easier.